helpers/dateParser.js

```diff
@@ -0,0 +1,13 @@
+// This function parses the date format from the user
+// user input query: /bookmarks?startDate=2018.12.01&endDate=2019.05.15
```

Comment:
this comment also doesn't match the current route

Reply:
Thank for that I have fixed the commenting!